src/app/card/info-status-card/examples/info-status-card-example.component.html

```diff
@@ -1,57 +1,47 @@
-<div class="padding-15">
-  <div class="row">
-    <div class="col-xs-12">
-      <h4>Info Card Component Example</h4>
-      <hr/>
-    </div>
+<div class="row">
+  <div class="col-xs-12">
+    <h4>Info Card Component Example</h4>
+    <hr/>
   </div>
-  <div class="cards-pf">
-    <div class="row row-cards-pf">
-      <div class="col-sm-4">
+</div>
+<div class="padding-15">
+  <div class="row row-cards-pf well">
+    <div style="max-width:409px;">
```

Comment:
This is conflicting with the column width that you have set below. In the "medium" viewport, you are making the card smaller than the column wants it to be.

I would delete this line, and change the column styles on the next. (see below)